py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
Ok this is all horrible.  We want to build the `format` and `format_device` from the old config which is hard because what was `format` is now `format_device` but we can tell if `{name}` is bing used in the format that it must be an old one.

I'd still prefer this PR to be cleaner especially as we are changing the behaviour (which can't be helped)

anyhow we are moving in the right direction